pkgs/development/libraries/frozen/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, writeText
+}:
+
+stdenv.mkDerivation rec {
+  pname = "frozen";
+  version = "unstable-2021-02-23";
```

Comment:
Frozen seems to have releases, do you think you can ask upstream to cut a new release to avoid having unstable versions for too long?

Reply:
Sure, I'll open an issue.

Reply:
https://github.com/cesanta/frozen/issues/72

Reply:
That's beyond the point, we just need to track it on the long run.